packages/blade/src/components/Chip/__tests__/ChipGroup.web.test.tsx

```diff
@@ -0,0 +1,268 @@
+import userEvents from '@testing-library/user-event';
+import React from 'react';
+import { Chip } from '../Chip';
+import type { ChipGroupProps } from '../ChipGroup/ChipGroup';
+import { ChipGroup } from '../ChipGroup/ChipGroup';
+import renderWithTheme from '~utils/testing/renderWithTheme.web';
+
+const selectionTypes: ChipGroupProps['selectionType'][] = ['single', 'multiple'];
+const sizes: ChipGroupProps['size'][] = ['xsmall', 'small', 'medium', 'large'];
+const intents: ChipGroupProps['intent'][] = ['neutral', 'positive', 'negative'];
+
+describe('<ChipGroup />', () => {
+  for (const selectionType of selectionTypes) {
+    describe(`selectionType="${selectionType}"`, () => {
```

Comment:
why do you need a describe here? you already have it for multiple behavior already?

Reply:
This is to provide a better output on the terminal.
<img width="513" alt="Screenshot 2023-08-28 at 8 20 48 AM" src="https://github.com/razorpay/blade/assets/46647141/51c6c31e-442c-42f0-ace6-c1bfb3a47738">